docs/tasks/timeout-injection.md

```diff
@@ -0,0 +1,116 @@
+# Injecting Timeouts
+
+![Timeout Injection](timeout-injection.png)
+
+Slow or unresponsive microservices can reveal defects in other system components.  Unhandled timeout
+exceptions can cascade and affect user experience.
+
+In this tutorial we will show advanced features of the Istio command line to inject timeouts into
+a running system.
+
+The Kubernetes-based implementation of Istio used in this tutorial works by injecting behavior
+into Pods.  All Pods that participate in the service mesh are enabled by the addition of a proxy.
+The _istioctl kube-inject_ subcommand can be used to apply the changes to a running Deployment.
+Later we will show how to modify the YAML files used to deploy your application so that it comes
+up immediately with the Istio mesh connection.
+
+## Before you begin
+
+This tutorial assumes you have deployed Istio on Kubernetes.  If you have not done so, please first
+follow [the installation instructions](../INSTALL.md).
```

Comment:
Pulling that out into a separate doc and referencing it from both places would be a good idea.